### PR TITLE
virt-manager: fix missing cdrtools

### LINKED ISSUE #124386: virt-install cannot find mkisofs
**Describe the bug**
invoking virt-install with the cloud-init images results in `ENOENT`

a workaround is `nix-shell -p cdrtools`

**To Reproduce**
1. run `virt-install --connect qemu:///system --memory 4000 --cloud-init root-password-generate=on --disk https://cloud.debian.org/images/cloud/bullseye/daily/20210525-648/debian-11-genericcloud-amd64-daily-20210525-648.qcow2`
2. `ERROR    [Errno 2] No such file or directory: 'mkisofs'`

**Expected behavior**
virt-install should proceed

**Screenshots**
NA

**Additional context**
It should probably be enough to add cdrtools to `makeBinPath`

**Notify maintainers**
@qknight @offlinehacker @fpletz @globin 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.37, NixOS, 21.05.20210525.bc71efa (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210503_6d2553a`
 - channels(root): `""`
 - channels(louis): `""`
 - nixpkgs: `/nix/store/df9dwmlgkx5br42g8md9nfcy6l0cbc57-source`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

### PR DESCRIPTION
**Pull request by superherointj**
Fixes #124386

### CODE REVIEW

**Comment by lourkeur** [comment]
Result of `nixpkgs-review pr 124408` run on x86_64-linux [1](https://github.com>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>virt-manager</li>
  </ul>
</details>

**Review by bbjubjub2494 — APPROVED** [review]
My issue is fixed in the resulting derivation.

File: pkgs/applications/virtualization/virt-manager/default.nix [hunk 1]
```diff
@@ -34,7 +34,7 @@ python3Packages.buildPythonApplication rec {
 
   propagatedBuildInputs = with python3Packages;
     [
```
Comment (SuperSandro2000) [inline pkgs/applications/virtualization/virt-manager/default.nix:36]:
```suggestion
  propagatedBuildInputs = with python3Packages; [
```

File: pkgs/applications/virtualization/virt-manager/default.nix [hunk 2]
```diff
@@ -34,7 +34,7 @@ python3Packages.buildPythonApplication rec {
...
-      pygobject3 ipaddress libvirt libxml2 requests
+      pygobject3 ipaddress libvirt libxml2 requests cdrtools
     ];
```
Comment (SuperSandro2000) [inline pkgs/applications/virtualization/virt-manager/default.nix:38]:
```suggestion
    pygobject3 ipaddress libvirt libxml2 requests cdrtools
  ];
```

**Pull request merged**